src/views/layouts/Header/Menu/__tests__/component.spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import MenuComponent from '../component'
+
+describe('MenuComponent', () => {
+  const defaultProps = {
+    onLogout: jest.fn()
+  }
+
+  it('matches snapshot', () => {
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
can we ingore this rule for all files with `*.spec.js` pattern?

Reply:
Good point, I try it